test/parallel/test-tls-hello-parser-failure.js

```diff
@@ -33,8 +34,8 @@ const net = require('net');
 const fs = require('fs');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/test_key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`)
+  key: fs.readFileSync(fixtures.path(`test_key.pem`)),
+  cert: fs.readFileSync(fixtures.path(`test_cert.pem`))
```

Comment:
There is the `fixtures.readKey` function to replace the `fs.readFileSync` call here. It would be nice to use that instead.

Reply:
ping @michaelwarren1106 ... when you get a moment, can you look at updating this to use the `fixtures.readKey()` method? For instance...

```js
key: fixtures.readKey('test_key.pem'),
cert: fixtures.readKey('test_cert.pem')
```

Reply:
the tests are now failing because those keys dont exist somehow.

thoughts?

Reply:
are the keys supposed to exist? i dont have anything in my tests/common/fixtures/keys directory called 'test_key.pem'.

So, I found that those pems aren't in the `keys` dir in fixtures like the fixtures module wants them, so I copied them over there so the tests pass again. Not sure I should move them, so for the time being there will be 2 copies of those pems.